tests/test_element_checker.py

```
@@ -1,36 +1,13 @@
-Created on:  4/25/24
-
-"""
+from typing import Any, Union
 
 import pytest
 
 from temoa.model_checking.element_checker import ViableSet, filter_elements
 
-params = [
+ParamType = dict[str, Any]
+ElementType = Union[tuple[Any, ...], str, int]
```

Ruff (0.14.4)

Use `X | Y` for type annotations

(UP007)